stacks/stack-platformsh.yml

```diff
@@ -0,0 +1,41 @@
+# Platform.sh alike stack
+# see https://github.com/platformsh-templates/drupal9
+#
+# - Nginx 1.14
+# - MariaDB 10.3
+# - PHP 7.4
+# - Redis 6.0
+
+version: "2.1"
+
+services:
+  # http(s)://VIRTUAL_HOST
+  web:
+    extends:
+      file: ${HOME}/.docksal/stacks/services.yml
+      service: nginx
+    # Pin nginx version
+    image: ${WEB_IMAGE:-docksal/nginx:1.14-edge}
+    depends_on:
+      - cli
+
+  db:
+    extends:
+      file: ${HOME}/.docksal/stacks/services.yml
+      service: mariadb
+    # Pin MariaDB version
+    image: ${DB_IMAGE:-docksal/mariadb:10.3-1.1}
+
+  cli:
+    extends:
+      file: ${HOME}/.docksal/stacks/services.yml
+      service: cli
+    # Pin PHP version
+    image: ${CLI_IMAGE:=docksal/cli:stable-php7.4}
+
+redis:
```

Comment:
Needs to be indented and aligned with the rest of the service entries.

Reply:
Oups, sorry about that, should be fixed now